bin/server.js

```diff
@@ -74,7 +74,14 @@ function serveIndex(req, res) {
 //
 app.get("/config.js", getConfigFile);
 
-app.get(/^\/[\w\-]+$/, serveIndex);
+var roomRouteRegexp = new RegExp(
+  "^\\/" + // starts with a /
+  "[" +
+    "\\w" + // non-whitespace characters
+    "\\-" + // - used in the room query sometimes (for crypto?)
```

Comment:
I think you can get room keys with "-" as well, but yes, its generally for the crypto stuff.